<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OFDM UI</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_setup">
       <attribute name="title">
        <string>Experiment Set Up</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="10" column="0">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Training Sequences</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QComboBox" name="modulation_type">
            <item>
             <property name="text">
              <string>QAM 4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>QAM 16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>QAM 64</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Sampling Rate (fs) [MHz]</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Modulation Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Tx Dev Identification</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="rx_dev_id"/>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mode">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>SISO (1x1)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rx Diversity (1x2)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rx Zero Forcing (2x2)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rx Beamforming (2x2)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tx Beamforming (2x2)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Number of Nullcarriers</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="tx_gain"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="fc"/>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="fs"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Center Frequency (fc) [MHz]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="tx_dev_id"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Rx Dev Identification</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Tx Gain</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="rx_gain"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Rx Gain</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Number of Subcarriers</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="training_sequences"/>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="cyclic_prefix_len"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <family>Abyssinica SIL</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Cyclic Prefix length</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="no_subcarriers">
            <property name="maxVisibleItems">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="no_nullcarriers"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="rub_logo">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/pics/logo_rub.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>Abyssinica SIL</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Log Output:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="log">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QCommandLinkButton" name="commandLinkButton">
                <property name="text">
                 <string>Stop Execution</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCommandLinkButton" name="run_button">
                <property name="text">
                 <string>Run Communication</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_constellation_1">
       <attribute name="title">
        <string>Constellation 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCustomPlot" name="const_tx_1" native="true"/>
          </item>
          <item>
           <widget class="QCustomPlot" name="const_rx_1" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_constellation_2">
       <attribute name="title">
        <string>Constellation 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCustomPlot" name="const_tx_2" native="true"/>
          </item>
          <item>
           <widget class="QCustomPlot" name="const_rx_2" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_signal_1">
       <attribute name="title">
        <string>Signal 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QCustomPlot" name="time_tx_1" native="true"/>
          </item>
          <item>
           <widget class="QCustomPlot" name="time_rx_1" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_signal_2">
       <attribute name="title">
        <string>Signal 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QCustomPlot" name="time_tx_2" native="true"/>
          </item>
          <item>
           <widget class="QCustomPlot" name="time_rx_2" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_response">
       <attribute name="title">
        <string>Channel Responses</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>177</y>
      <width>179</width>
      <height>75</height>
     </rect>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mode</tabstop>
  <tabstop>tx_dev_id</tabstop>
  <tabstop>rx_dev_id</tabstop>
  <tabstop>tx_gain</tabstop>
  <tabstop>rx_gain</tabstop>
  <tabstop>fc</tabstop>
  <tabstop>fs</tabstop>
  <tabstop>cyclic_prefix_len</tabstop>
  <tabstop>training_sequences</tabstop>
  <tabstop>modulation_type</tabstop>
  <tabstop>run_button</tabstop>
  <tabstop>commandLinkButton</tabstop>
  <tabstop>tabs</tabstop>
  <tabstop>log</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
